graspy/classify/vertex_screen.py

```diff
@@ -0,0 +1,80 @@
+from .base import BaseSignalSubgraph
+from ..utils import import_graph, is_symmetric
+from mgcpy.independence_tests.dcorr import DCorr
+from mgcpy.independence_tests.mgc import MGC
+import numpy as np
+
+
+class VertexScreener(BaseSignalSubgraph):
+    def __init__(self, num_vertices=20, distance_metric="mgc"):
+        self.distance_metric = distance_metric
+        self.num_vertices = num_vertices
+
+    def fit(self, graphs, y):
+        self._vertices, self._ts = self._screen_vertices(graphs, y)
```

Comment:
Some type checks? E.g. make sure size of graphs and y match, graph is actually a graph (e.g. square, numeric, etc) - you can use import_graph here, I believe `mgc` requires `y` to be a 2d array.. Im sure I missed other checks, but think about other checks here.